deployment/k8sPaiLibrary/maintainlib/update.py

```diff
@@ -84,16 +86,15 @@ def get_node_list_from_k8s_api(self):
     def get_node_config_from_cluster_configuration(self):
 
         self.logger.info("Try to get node confguration from cluster configuration.")
-        cluster_config = self.k8s_configuration
+        com = self.k8s_configuration
```

Comment:
And here, why we need a variable `com`?

Reply:
Get node configuration to constuct node list. And BTW, this change is introduced before and not in this PR.